src/backend/distributed/shared_library_init.c

```diff
@@ -356,6 +356,16 @@ RegisterCitusConfigVariables(void)
 		0,
 		NULL, NULL, NULL);
 
+	DefineCustomBoolVariable(
+		"citus.enable_router_planner",
```

Comment:
I'm not really following. The GUCs naming doesn't have to change behaviour.  But anyway, I don't think it's that important.